Wikipedia/Code/WMFNearbyArticleTableViewCell.m

```diff
@@ -197,4 +197,20 @@ - (void)setImage:(MWKImage*)image {
     [self.articleImageView wmf_setImageWithMetadata:image detectFaces:YES];
 }
 
+#pragma mark - Accessibility
+
+- (BOOL)isAccessibilityElement {
+    return YES;
+}
+
+- (NSString*)accessibilityLabel {
+    NSString *titleAndDescription;
+    if (self.descriptionText) {
+        titleAndDescription = [NSString stringWithFormat:@"%@, %@", self.titleText, self.descriptionText];
+    } else {
+        titleAndDescription = self.titleText;
+    }
+    return [NSString stringWithFormat: @"%@, %@ %@", titleAndDescription, self.distanceLabel.text, self.compassView.accessibilityLabel];
```

Comment:
maybe a minor point, but shouldn't the accessibility label for the distance say the full unit name, i.e. meters/feet instead of m/ft?